components/basecomponents/KidsTranslationContainer.tsx

```diff
@@ -1,53 +1,86 @@
 import React from 'react';
-import Image from 'next/legacy/image';
+import { useAtomValue } from 'jotai';
+
+/** Components */
 import { KidsTranslation } from './KidsTranslation';
+import KioskDictionaryCardImage from './KioskDictionaryGame/KioskDictionaryCardImage';
+
+/** Hooks, Types, Utils, etc. */
 import { useLanguage } from 'utils/useLanguageHook';
-import { AudioPlayer } from 'utils/AudioPlayer';
-import { useTranslation } from 'next-i18next';
-import { Phrase } from '../../utils/getDataUtils';
-
-interface KidsTranslationContainerProps {
-  phrase: Phrase;
-  imageUrl: string | null;
-  id?: string;
-  searchText?: string;
-}
-
-/**
- *  Displays list of translations in opened collapse component
- *
- * @returns
- */
-export const KidsTranslationsContainer = ({ phrase, imageUrl, id }: KidsTranslationContainerProps): JSX.Element => {
+import { Platform, KidsTranslationContainerProps } from '@types';
+import { currentPlatformAtom, dictionaryAudioPlayAtom, dictionaryActivePhraseAtom } from './Kiosk/atoms';
+import { Language, LanguageEnum } from 'utils/locales';
+
+const ACTIVE_STYLE = {
+  [LanguageEnum.CZECH]: 'czech-sound !bg-kiosk-red',
+  [LanguageEnum.UKRAINIAN]: 'ukraine-sound !bg-kiosk-yellow',
+};
+
+const KidsTranslationsContainer = ({ phrase, imageUrl, id }: KidsTranslationContainerProps): JSX.Element => {
   const { currentLanguage, otherLanguage } = useLanguage();
-  const { t } = useTranslation();
+  const renderFor = useAtomValue<Platform>(currentPlatformAtom);
+  const isPlaying = useAtomValue(dictionaryAudioPlayAtom);
+  const activePhrase = useAtomValue(dictionaryActivePhraseAtom);
+
+  const isActiveLanguage = (language: Language) =>
+    isPlaying && activePhrase === phrase.getTranslation(language) && renderFor === Platform.KIOSK;
 
-  const currentTranslation = phrase.getTranslation(currentLanguage);
-  const otherTranslation = phrase.getTranslation(otherLanguage);
+  const isActive = {
+    [LanguageEnum.CZECH]: isActiveLanguage(LanguageEnum.CZECH),
```

Comment:
Why is this only checking Czech an Ukrainian? Is this component not used in Slovak an Polish variations as well?

Reply:
In this phase, we will support Czech and Ukraine only.

Reply:
Okay, but this component is also used in the regular non-kiosk kids dictionary right?

Reply:
Dik, fixnute